.github/workflows/ci.yml

```diff
@@ -69,10 +69,11 @@ jobs:
           choco install wget --no-progress
 
           if wget https://github.com/TileDB-Inc/TileDB/releases/download/$LIBTILEDB_VERSION/tiledb-windows-x86_64-$LIBTILEDB_VERSION-$LIBTILEDB_SHA.zip; then
-            unzip tiledb-windows-x86_64-$LIBTILEDB_VERSION-$LIBTILEDB_SHA.zip -d /usr/local
-            cp /usr/local/bin/tiledb.dll tiledb
-            python setup.py develop --tiledb=/usr/local
-          else 
+            mkdir libtiledb
```

Comment:
For future reference, this change is to work around `Can't restore time` errors. x-ref https://app.shortcut.com/tiledb-inc/story/41044 for follow-up.